spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaMessageChannelBinder.java

```diff
@@ -252,9 +256,11 @@ public void stop(Runnable callback) {
 					}
 				};
 		messageListenerContainer.setConcurrency(concurrency);
-		messageListenerContainer.getContainerProperties().setAckOnError(isAutoCommitOnError(extendedConsumerProperties));
+		messageListenerContainer.getContainerProperties()
+				.setAckOnError(isAutoCommitOnError(extendedConsumerProperties));
```

Comment:
Not related to this change, but since I have noticed the fact, I decided to comment.

There is a PR in Spring Kafka https://github.com/spring-projects/spring-kafka/pull/293 which purpose to disable `ackOnError` if the mode is MANUAL.
Something similar is said on the appropriate JavaDocs:
```
* Set whether or not the container should commit offsets (ack messages) where the
 * listener throws exceptions. This works in conjunction with {@link #ackMode} and is
 * effective only when the kafka property {@code enable.auto.commit} is {@code false};
 * it is not applicable to manual ack modes. 
```